src/applications/edu-benefits/feedback-tool/config/form.js

```diff
@@ -59,7 +59,11 @@ const {
 } = fullSchema.properties;
 
 const { assistance, programs, school } = educationDetails.properties;
-const { address: schoolAddress, name: schoolName } = school.properties;
+const {
+  address: schoolAddress,
+  name: schoolName,
+  facilityCode,
+} = school.properties;
```

Comment:
okay, so I guess the drawback of Prettier is that is wrapped this block adding more noise to this particular diff :(

Reply:
net gain